ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/IntTopkKudafTest.java

```diff
@@ -176,19 +180,20 @@ public void testMergePerformance() {
     topkKudaf = new TopKAggregateFunctionFactory(topX)
         .getProperAggregateFunction(Collections.singletonList(Schema.INT32_SCHEMA));
 
-    final Integer[] aggregate1 = IntStream.range(0, topX)
+    final List<Integer> aggregate1 = IntStream.range(0, topX)
         .mapToObj(v -> v % 2 == 0 ? v + 1 : v)
-        .toArray(Integer[]::new);
-    final Integer[] aggregate2 = IntStream.range(0, topX)
+        .collect(Collectors.toList());
+    final List<Integer> aggregate2 = IntStream.range(0, topX)
         .mapToObj(v -> v % 2 == 0 ? v : v + 1)
-        .toArray(Integer[]::new);
+        .collect(Collectors.toList());
     final long start = System.currentTimeMillis();
 
     for(int i = 0; i != iterations; ++i) {
-      topkKudaf.getMerger().apply("ignmored", aggregate1, aggregate2);
+      topkKudaf.getMerger().apply("ignmored", new ArrayList<>(aggregate1), new ArrayList<>
```

Comment:
convert to:

```
topkKudaf.getMerger().apply("ignmored", aggregate1, aggregate2);
```

To avoid array copies inside the timed loops and hence skewing the results.